package.json

```diff
@@ -87,5 +87,8 @@
   },
   "ember-addon": {
     "configPath": "tests/dummy/config"
-  }
+  },
+  "resolutions": {
+  "ansi-html": "0.0.8"
```

Comment:
I think this resolution can now be removed and the lock file will stay resolved at `0.0.8` given it's set to `^0.0.7`

Reply:
Yeah, I was playing around with that. It goes back to using `0.0.7` when I remove the resolutions. I can't figure out why though. Both the dependencies specify `^0.0.7` 🤨

Reply:
Ah yes, this is because the `^` behaves differently for prerelease versions `0.x`. Add the range and package here and it confirms that it will always resolve to `0.0.7` https://semver.npmjs.com/

Let's leave the resolutions in.

Reply:
Haha oh yeah, I did notice that just before when deleting the resolutions 😅 I'll fix it up